tests/HJSONParserTest.php

```diff
@@ -10,7 +10,7 @@
 class HJSONParserTest extends TestCase
 {
 
-    public function setUp()
+    protected function setUp(): void
```

Comment:
If using the `:void` type hint, it should let this package require `php-7.1` version at least.

Reply:
So the latest version of phpunit (8.0.2) has strict typing and requires this type annotation. Also, even the last-supported version of phpunit doesn't support older than PHP 7.1: https://phpunit.de/announcements/phpunit-7.html. I've made an issue about dropping support for older PHP versions at #20.

Reply:
It makes sense, but it need to set the required PHP version in `composer.json`.

I think we should write the new released draft when make the latest release.

Reply:
I suppose this should really be in a separate pull request anyway.

Reply:
@dqsully, sure.

And it should let the Travis CI build be passed firstly.